invenio_cli/helpers/scripts.py

```diff
@@ -101,17 +101,32 @@ def bootstrap(log_config, dev=True, pre=True,
 
 
 @with_appcontext
-def build_assets(log_config):
-    """Build assets."""
+def _build_local_assets(log_config, statics=True, webpack=True):
+    """Build assets in the containers."""
     cli = create_cli()
     runner = current_app.test_cli_runner()
-    # Collect
-    run_command(cli, runner, "collect -v", message="Collecting assets...",
-                verbose=log_config.verbose)
+    if statics:
+        # Collect
+        run_command(cli, runner, "collect -v", message="Collecting assets...",
+                    verbose=log_config.verbose)
+    if webpack:
+        # Build using webpack
+        run_command(cli, runner, "webpack buildall",
+                    message="Building assets...", verbose=log_config.verbose)
+
+
+def _build_container_assets(log_config, statics=True, webpack=True):
+    """Build assets in the containers."""
+    # TODO: Check how to address this. Requires uwsgi restart.
+    pass
```

Comment:
Normal: Let's display a message that this is not something to do in prod / a full prod rebuild is what will take these changes into account. Important part here is to provide feedback to the user as opposed to no feedback.

Reply:
True :) fixed. Example output:
![image](https://user-images.githubusercontent.com/6756943/70517952-c6c35f80-1b39-11ea-9b6b-81e9f4135620.png)

Reply:
Nice!